packages/cms-lib/lib/scopes.js

```diff
@@ -0,0 +1,30 @@
+const { logger } = require('../logger');
+const { SCOPE_GROUPS } = require('./constants.js');
+const { fetchScopeData } = require('../api/localDevAuth/authenticated');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('@hubspot/cms-lib/errorHandlers');
+
+/**
+ *
+ * @param {number} portalId
+ */
+async function getScopeDataForFunctions(portalId) {
```

Comment:
Not something we need to do now, but seeing this makes me wonder what would need to change in order for the BE to return the missing scope group so that we could reuse the scope related enhanced error handling across all the places where we're using v3 endpoints.